### PR TITLE
refactor: library e2e

### PR DESCRIPTION
**Pull request by jrasm91**
With jest `12.572 s`
With vitest `9332ms`
With vitest without the beforeEach `440ms`

### CODE REVIEW

File: e2e/src/api/specs/library.e2e-spec.ts [hunk 1]
```diff
@@ -467,7 +454,7 @@ describe(`${LibraryController.name} (e2e)`, () => {
         expect(pathResponse).toEqual({
           importPath: pathToTest,
           isValid: false,
-          message: `Path does not exist (ENOENT)`,
+          message: `Not a directory`,
```
Comment (jrasm91) [inline e2e/src/api/specs/library.e2e-spec.ts:457]: It looks like this nest was never actually implemented so I've gone ahead and fixed it.

File: e2e/vitest.config.ts [hunk 2]
```diff
@@ -1,9 +1,17 @@
 import { defineConfig } from 'vitest/config';
 
+// skip `docker compose up` if `make e2e` was already run
+const globalSetup: string[] = [];
```
Comment (jrasm91) [inline e2e/vitest.config.ts:4]: Added this in here as well.

**Review by etnoy — APPROVED** [review]

**Comment by etnoy** [comment]
Nice work!

**Review by danieldietzler — COMMENTED** [review]
Dang you literally made it 28x faster! :O

**Pull request merged**